tests/test_db_tools.py

```diff
@@ -6,21 +6,68 @@
 class TestDBTools(unittest.TestCase):
 
     def setUp(self):
-        self.database = DatabaseConnector(os.getenv('DATABASE_URL_TEST'))
+        """Assuming the database is already set up."""
+        url = os.getenv('DATABASE_URL_TEST')
+        self.assertTrue(url)
+        if not url:
+            return
+        self.database = DatabaseConnector(url)
+        mock_article = {
+            'title': 'foo',
+            'uri': 'http://foo.bar',
```

Comment:
can't believe this URL is unregistered

Reply:
It's local anyway. (Is there a `.bar` domain? I need to know, it's... for a friend)